README.md

```diff
@@ -36,54 +47,61 @@ and stability.
-- `display_setting`
-  - __Type__: `Dict`
-  - __Description__: Set display setting for concrete model
-  - __Default__: `{}`
-  - __Options__: `{'app.model': 'display_default'}`
-  - __Example__: `{'dcim.device': 'full_width_page', 'virtualization.virtualmachine': 'right_page'}`
-- `enable_contacts`
-  - __Type__: `Boolean`
-  - __Description__: Enable contacts for secret
-  - __Default__: `False`
-- `public_key_size`
-  - __Type__: `Integer`
-  - __Description__: Size of the public key
-  - __Default__: `2048`
-  - __Options__: `2048`, `4096`, `8192`
-- `top_level_menu`
-  - __Type__: `Boolean`
-  - __Description__: Enable top level menu
-  - __Default__: `False`
+* `apps`
+  * __Type__: `List`
+  * __Description__: List of apps to enable
+  * __Default__: `['dcim.device', 'virtualization.virtualmachine']`
+* `display_default`
+  * __Type__: `String`
+  * __Description__: Where to display the secret on the detail page of the defined apps
+  * __Default__: `left_page`
+  * __Options__: `left_page`, `right_page`, `full_width_page`, `tab_view`
+* `display_setting`
+  * __Type__: `Dict`
+  * __Description__: Set display setting for concrete model
+  * __Default__: `{}`
+  * __Options__: `{'app.model': 'display_default'}`
+  * __Example__: `{'dcim.device': 'full_width_page', 'virtualization.virtualmachine': 'right_page'}`
+* `enable_contacts`
+  * __Type__: `Boolean`
+  * __Description__: Enable contacts for secret
+  * __Default__: `False`
+* `public_key_size`
+  * __Type__: `Integer`
+  * __Description__: Size of the public key
+  * __Default__: `2048`
+  * __Options__: `2048`, `4096`, `8192`
+* `top_level_menu`
+  * __Type__: `Boolean`
+  * __Description__: Enable top level menu
+  * __Default__: `False`
 
 ## Extra Configuration
 
 The following options are inherited from NetBox to configure the cookies:
 
-- `SESSION_COOKIE_SECURE`
-  - __Type__: `Boolean`
-  - __Description__: [Session Cookie Secure](https://docs.netbox.dev/en/stable/configuration/security/#session_cookie_secure)
-- `LOGIN_TIMEOUT`
-  - __Type__: `Integer`
-  - __Description__: [Login Timeout](https://docs.netbox.dev/en/stable/configuration/security/#login_timeout)
+* `SESSION_COOKIE_SECURE`
+  * __Type__: `Boolean`
+  * __Description__: [Session Cookie Secure](https://docs.netbox.dev/en/stable/configuration/security/#session_cookie_secure)
+* `LOGIN_TIMEOUT`
+  * __Type__: `Integer`
+  * __Description__: [Login Timeout](https://docs.netbox.dev/en/stable/configuration/security/#login_timeout)
 
 __Note: These options should be set in the NetBox configuration file.__
 
+## User Keys
+
+Each user within NetBox can associate his or her account with an RSA public key. If activated by an administrator, this user key will contain a unique, encrypted copy of the AES master key needed to retrieve secret data.
+
+User keys may be created by users individually, however they are of no use until they have been activated by a user who already possesses an active user key.
```

Comment:
This sounds a bit redundant to me. It's better to rephrase it

Reply:
This was copied from `docs/models/userkey.md`. I personally Prefer every usage or deployment related inside the main `Readme.md` file and only development documentation inside the `docs` directory.

I will bring everything into the `Readme.md` file and you tell me what you think of it